Formula/musikcube.rb

```diff
@@ -0,0 +1,40 @@
+class Musikcube < Formula
+  desc "Terminal-based audio engine, library, player and server"
+  homepage "https://musikcube.com"
+  url "https://github.com/clangen/musikcube/archive/0.99.3.tar.gz"
+  sha256 "584c8225caad46fde83cdd520ef7d4a6633471de9693651f46d520040e9fb51d"
+  license "BSD-3-Clause"
+  head "https://github.com/clangen/musikcube.git", branch: "master"
+
+  depends_on "ccache" => :build
+  depends_on "cmake" => :build
+  depends_on "asio"
+  depends_on "curl"
+  depends_on "ffmpeg"
+  depends_on "game-music-emu"
+  depends_on "gnutls"
+  depends_on "lame"
+  depends_on "libev"
+  depends_on "libmicrohttpd"
+  depends_on "libogg"
+  depends_on "libopenmpt"
+  depends_on "libvorbis"
+  depends_on "ncurses"
+  depends_on "openssl@1.1"
+  depends_on "taglib"
+
+  def install
+    args = std_cmake_args + %w[
+      -DCMAKE_BUILD_TYPE=Release
+    ]
+    system "cmake", ".", *args
+    system "make"
+    system "cmake", "."
+    system "make", "install"
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

Reply:
`musikcube` has a somewhat complicated build process due to (1) the number of operating systems and versions it supports, and (2) the dynamic nature of the plugin system. 

If at all possible I'd prefer to keep the precise sequence and contents of the `cmake` commands, as they are used to build every variant across every platform. Is this suggested change non-negotiable?

Reply:
https://github.com/Homebrew/homebrew-core/pull/118938/commits/b344ea930578a7cf202ca82762ab85bfd399f396